unit-tests.sh

```diff
@@ -15,11 +15,12 @@
 
 
 CONFIG_FILE=config-devel.toml
-POSTGRES_PORT=$(cat $CONFIG_FILE | grep -i pg_port | grep -E -o '[0-9]+')
+POSTGRES_PORT=$(grep -i pg_port $CONFIG_FILE | grep -E -o '[0-9]+')
 
 function run_unit_tests() {
-    go test -coverprofile coverage.out $(go list ./... | grep -v tests)
-    if [ $? -ne 0 ]; then
+    # shellcheck disable=SC2046
```

Comment:
I was totally unable to resolve this problem without breaking tests execution, so I disabled this warning